src/icons/general/IconPuzzlePiece.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    dimension?: number,
+    title?: string,
+};
+
+const IconPuzzlePiece = ({ className = '', color = '#AFAFAF', dimension = 14, title }: Props) => (
```

Comment:
Can you add `#AFAFAF` to variables.js, and then update here and IconSearch.js?

Reply:
Is there a naming convention for these variables?

Reply:
It looks like it already exists as `silverChalice` 🍷